server/src/main/kotlin/com/mobilejazz/colloc/feature/encoder/AndroidLocalizationEncoder.kt

```diff
@@ -0,0 +1,54 @@
+package com.mobilejazz.colloc.feature.encoder
+
+import com.mobilejazz.colloc.domain.model.Dictionary
+import com.mobilejazz.colloc.domain.model.Language
+import com.mobilejazz.colloc.util.Android.encodeLiterals
+
+class AndroidLocalizationEncoder : LocalizationEncoder {
+
+  lateinit var importedUrlLocation: String
+
+  override fun encodeContent(dictionary: Dictionary): Map<Language, String> =
+    dictionary
+      .mapValues {
+        it.value.encode()
+      }
+
+  override fun encodeLocalizationFileName(language: Language): String = "/values${language.encodeLanguageCode()}/strings.xml"
+
+  private fun Map<String, String>.encode(): String =
+    buildString {
+      append("<?xml version=\"1.0\" encoding=\"utf-8\"?>\n")
+      append("<resources>\n\n")
+      append(
+        "<!--\r\n\tThis is an automatically generated file\r\n\tPlease don't modify it.\r\n\t" +
+          "If you need to change some text please do so at\r\n\t$importedUrlLocation\r\n\t-->\r\n\r\n"
```

Comment:
Please, remove the `$importedUrlLocation` label because it does not add a lot of value and complicates the code.